bindings/cross_l2_inbox.go

```diff
@@ -0,0 +1,65 @@
+package bindings
```

Comment:
not for now but for a future PR we should just include the abigen generated bindings instead of a custom one like this

Reply:
yea i agree. I was going down that path but left that off till later since it's not the most important